src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -16,9 +16,8 @@
  * Provide default OG selection handler.
  *
  * @EntityReferenceSelection(
- *   id = "default:og",
+ *   id = "og:default",
```

Comment:
Is this intentional? If I look in a similar class like `\Drupal\taxonomy\Plugin\EntityReferenceSelection\TermSelection` then they also use `default:taxonomy_term` and not `taxonomy_term:default`.